src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1631,16 +1761,19 @@ bool processOnChangeCondition
   std::string          err;
   NotifyContextRequest ncr;
 
-    //
     // FIXME P10: we are using dummy scope at the moment, until subscription scopes get implemented
     //
     Restriction res;    
-    if (!entitiesQuery(enV, attrL, res, &ncr.contextElementResponseVector, &err, false, tenant, servicePathV))
+    ContextElementResponseVector rawCerV;
+    if (!entitiesQuery(enV, attrL, res, &rawCerV, &err, false, tenant, servicePathV))
     {
         ncr.contextElementResponseVector.release();
         return false;
     }
 
+    /* Prune "not found" CERs */
+    pruneNotFoundContextElements(rawCerV, &ncr.contextElementResponseVector);
```

Comment:
Isn't *pruneContextElements* a good enough name?
I mean, the semantics of 'prune' already implies 'not found', right?